CMakeLists.txt

```diff
@@ -615,14 +616,29 @@ psibase_package(
     DEPENDS ${InvitesPlugin_DEP}
 )
 
+psibase_package(
+    OUTPUT ${SERVICE_DIR}/Email.psi
+    NAME Email
```

Comment:
I don't think we should call this "email" unless it's actually able to interface to SMTP somehow. Otherwise, anyone seeing the name is just going to be confused.

Reply:
How about "Webmail"?

Reply:
Renamed it webmail. If you want yet another name we can align later and I'll make a separate pr